src/Relations/MorphMany.php

```diff
@@ -14,13 +15,8 @@
  */
 class MorphMany extends EloquentMorphMany
 {
-    /**
-     * Get the name of the "where in" method for eager loading.
-     *
-     * @param string $key
-     *
-     * @return string
-     */
+    /** Get the name of the "where in" method for eager loading. */
```

Comment:
Same here, the comment can be removed entirely. https://github.com/laravel/framework/blob/1831e267177acf34fabd121fc0b7eaf32d61c06d/src/Illuminate/Database/Eloquent/Relations/Relation.php#L423